packages/cli/src/commands/experimental/setupRealtimeHandler.js

```diff
@@ -0,0 +1,291 @@
+import fs from 'fs'
+import path from 'path'
+
+import { Listr } from 'listr2'
+
+import { generate as generateTypes } from '@redwoodjs/internal/dist/generate/generate'
+import { getConfigPath } from '@redwoodjs/project-config'
+import { errorTelemetry } from '@redwoodjs/telemetry'
+
+import { getPaths, transformTSToJS, writeFile } from '../../lib'
+import c from '../../lib/colors'
+import { isTypeScriptProject } from '../../lib/project'
+
+import { command, description, EXPERIMENTAL_TOPIC_ID } from './setupRealtime'
+import { printTaskEpilogue, isServerFileSetup } from './util'
+
+export async function handler({ force, verbose }) {
+  const redwoodPaths = getPaths()
+  const ts = isTypeScriptProject()
+
+  const realtimeLibFilePath = path.join(
+    redwoodPaths.api.lib,
+    `realtime.${isTypeScriptProject() ? 'ts' : 'js'}`
+  )
+
+  const tasks = new Listr(
+    [
+      {
+        title: 'Confirmation',
+        task: async (_ctx, task) => {
+          const confirmation = await task.prompt({
+            type: 'Confirm',
+            message:
+              'Realtime is currently an experimental RedwoodJS feature. Continue?',
+          })
+
+          if (!confirmation) {
+            throw new Error('User aborted')
+          }
+        },
+      },
+      {
+        title: 'Checking for realtime environment prerequisites ...',
+        task: () => {
+          isServerFileSetup()
+        },
```

Comment:
What would you think about adding a task here that replaces the commented out strings in the updated `server.ts` template with the uncommented version?

I know the comments are just temporary while this is all experimental and subject to change but it might not be that bad to add a few "replace one strings with another" operations in a task here?

Reply:
It probably should, but I think a codemod might be better here; I just didn't get around to writing one as was going to wait for the final templates to settle down.